sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -812,13 +812,6 @@ def test_neighbors_badargs():
                       cls, p=-1)
         assert_raises(ValueError,
                       cls, algorithm='blah')
-        nbrs = cls(algorithm='ball_tree', metric='haversine')
```

Comment:
Sorry this was my suggestion.

> So you got "error not raised" here?

Yes. Travis failed owing to this. I assumed since haversine was not previously implemented, this test was introduced.